packages/nimble-components/src/banner/tests/banner.stories.ts

```diff
@@ -0,0 +1,130 @@
+`;
+
+const metadata: Meta<BannerArgs> = {
+    title: 'Banner',
+    decorators: [withXD],
+    parameters: {
+        docs: {
+            description: {
+                component: overviewText
+            }
+        },
+        design: {
+            artboardUrl:
+                'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/7b53bb3e-439b-4f13-9d5f-55adc7da8a2e/specs/'
+        },
+        actions: {
+            handles: ['toggle']
+        }
+    }
+};
+
+export default metadata;
+
+export const _banner: StoryObj<BannerArgs> = {
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html`
+        <nimble-banner
+            ?open="${x => x.open}"
+            severity="${x => x.severity}"
+            ?title-hidden="${x => x.titleHidden}"
+            ?prevent-dismiss="${x => x.preventDismiss}"
+            dismiss-button-label="Close"
+        >
+            <span slot="title">${x => x.title}</span>
+            ${x => x.text}
+            ${when(x => x.action === 'button (ghost)', html`
+                <nimble-button slot="action" appearance="ghost">Do action</nimble-button>`)}
+            ${when(x => x.action === 'button (outline)', html`
+                <nimble-button slot="action" appearance="outline">Do action</nimble-button>`)}
+            ${when(x => x.action === 'anchor', html`
+                <nimble-anchor slot="action" href="#">Go to site</nimble-anchor>`)}
+        </nimble-banner>
+`),
+    // eslint-disable-next-line storybook/no-redundant-story-name
+    storyName: 'Banner',
+    argTypes: {
+        open: {
+            description: 'Controls whether the banner is visible.'
+        },
+        title: {
+            description:
+                'Primary text which is displayed when `title-hidden` is not set. Banners should **always include a title** even when `title-hidden` is set. The title is used to provide an accessible name to assistive technologies regardless of the value of `title-hidden`.<br><br>Provide the title in an `inline` element such as `<span>` that is targeted to the `title` slot.'
+        },
+        text: {
+            description:
+                'Secondary text that provides a detailed message. This text should be short enough to fit within three lines when displayed.<br><br>Provide this text as content of the banner element (uses the default slot).'
+        },
+        severity: {
+            options: Object.values(BannerSeverity),
+            control: { type: 'radio' },
+            description:
+                'Severity of the message presented by the banner. Controls the displayed color and icon. If not set, the banner has a neutral appearance.'
+        },
+        action: {
+            options: Object.values(ActionType),
```

Comment:
Same here -- this should use `Object.keys(ActionType)`

Reply:
Talked to you about this. I want to use `Object.values` so that I can have labels like "button (ghost)" instead of "buttonGhost".